README.md

```diff
@@ -0,0 +1,106 @@
+# PersonalFinanceTool
+
+## Introduction
+
+The PersonalFinanceTool (PFT) is an application which allows a user to keep track of the total amount of money they have available as well as bills which they wish to track. Bills will be tracked in the form of name, amount due, and due date. The user can allocate funds from their total amount available to their tracked bills.
+
+## Prerequisities
+
+Before attempting to utilize the PFT, a user should ensure they have the following installed on their machine:
```

Comment:
This section is not very detailed. Perhaps walk the reader through the process of installing development tools and Qt on their machine. This may need to go into a separate markdown or LaTeX file.

Reply:
Added additional detail. Not a bad idea to expand even more in a later version in a separate .md file or in a LaTeX document.